### PR TITLE
connect to api

### PR DESCRIPTION
**Pull request by AhmedNazirMusah**
On this milestone:
- Used fetch and Axios with createAsyncThunk for making API calls.
- Fetch book list from Bookstore API on Books page load.
- Added an Action Creator, that returns a function.
- Fetch the list of books.
- Dispatched another action that will pass the API response to the reducer and update the state.
- Refactored my create book and remove book features to persist your changes in the server.
- Changed your existing Action Creators, so they return functions.
- Made changes by using API.

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { configureStore } from '@reduxjs/toolkit';
+// import thunk from 'redux-thunk';
```
Comment (1GORDON) [inline src/redux/configureStore.js:2]: Great job @AhmedNazirMusah 🤝

- [ ] [Optional] You may always remove commented code from your project as this is regarded as a bad practice.

File: src/components/Books.js [hunk 2]
```diff
@@ -9,3 +9,5 @@ const Books = () => (
 );
 
 export default Books;
+
+// ZMP8Cx3pL0HB31pM9poX
```
Comment (1GORDON) [inline src/components/Books.js:13]: Great job @AhmedNazirMusah 🤝

- [ ] [Optional] You may always remove commented code from your project as this is regarded as a bad practice.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @AhmedNazirMusah ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 2 above)
Comment (AhmedNazirMusah) [reply src/components/Books.js:13]: Thanks for drawing my attention to that, thank you 😃 .

**Pull request merged**